crates/bevy_transform/src/lib.rs

```diff
@@ -76,17 +77,15 @@ impl From<Transform> for TransformBundle {
     }
 }
 /// Label enum for the systems relating to transform propagation
-#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemLabel)]
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
 pub enum TransformSystem {
     /// Propagates changes in transform to children's [`GlobalTransform`](crate::components::GlobalTransform)
     TransformPropagate,
 }
 
-/// Transform propagation system set for third party plugins use
-pub fn transform_propagate_system_set() -> SystemSet {
-    SystemSet::new()
-        .with_system(systems::sync_simple_transforms)
-        .with_system(systems::propagate_transforms)
+/// A [`Schedule`] that contains correctly ordered transform propagation systems for third party plugin use
+pub fn transform_propagate_systems() -> SystemConfigs {
+    IntoSystemConfigs::into_configs((sync_simple_transforms, propagate_transforms))
 }
```

Comment:
bevy_rapier also [exports SystemSets](https://github.com/dimforge/bevy_rapier/blob/3899db0e4828309fe2c7f44677a080f4625c68d8/src/plugin/plugin.rs#L58)(SystemConfigs) which includes transform propagation.
This isn't possible now that transform propagation is exported only as a `SystemConfigs` because you can't 'nest' `SystemConfigs`.
Could we make `SystemConfigs` more flexible/composable?